apps/aecore/test/aec_chain_tests.erl

```diff
@@ -782,6 +783,50 @@ fork_get_transaction() ->
 
     ok.
 
+fork_on_micro_block() ->
+    #{ public := PubKey, secret := PrivKey } = enacl:sign_keypair(),
+    PresetAccounts = [{PubKey, 1000}],
+    meck:expect(aec_genesis_block_settings, preset_accounts, 0, PresetAccounts),
+
+    %% Create main chain with both key and micro blocks
+    TxsFun = fun(2) ->
+                     Tx1 = aec_test_utils:sign_tx(make_spend_tx(PubKey, 1, PubKey, 1), PrivKey),
+                     Tx2 = aec_test_utils:sign_tx(make_spend_tx(PubKey, 2, PubKey, 1), PrivKey),
+                     [Tx1, Tx2];
+                (_) ->
+                     []
+             end,
+    [B0, B1, B2, _, _, _] =
+        Chain0 = gen_block_chain_with_state_by_target(
+                   PresetAccounts,
+                   [?HIGHEST_TARGET_SCI, ?HIGHEST_TARGET_SCI, ?HIGHEST_TARGET_SCI], 1, TxsFun),
+    [KB0, KB1, MB1, MB2, KB2, KB3] = blocks_only_chain(Chain0),
```

Comment:
Assertion that in this generated test date actually the KB* are key blocks and the MB* are micro blocks would help readability of test (e.g. not sure how microblocks are 2 and not 1 - with 2 txs).